src/interface.jl

```diff
@@ -0,0 +1,82 @@
+"""
+  istree(x)
```

Comment:
```suggestion
    istree(x)
```
A 4-space indent is needed so that the signature is printed as Julia code in the documentation.
The same applies to other functions in this file.